packages/lib/src/components/PayByBankUS/PayByBankUS.scss

```diff
@@ -0,0 +1,20 @@
+@import 'styles/variable-generator';
+
+.adyen-checkout-paybybank_AIS_DD {
+    margin-bottom: token(spacer-070);
+
+    &__description-header {
+        margin: token(spacer-000) token(spacer-000) token(spacer-020);
+        font-size: token(text-subtitle-font-size);
+        font-weight: token(text-body-stronger-font-weight);
+        color: token(color-label-primary);
+    }
+
+    &__description-body {
+        list-style-type: disc;
+        color: token(color-label-secondary);
+        margin: 0;
+        font-size: token(text-body-font-size);
+        line-height: 1.5;
+    }
```

Comment:
The `font-weight` of the body needs to be set to `400` - right now it is bold. 

And there is some margin pixels missing between the header and body - the header should push the body down by 6px.

(I checked with Arjen)

Reply:
How do you mean it's bold? It isn't bold?

Reply:
![Screenshot 2024-09-18 at 14 24 55](https://github.com/user-attachments/assets/70a619f4-9a2d-41b2-9008-e63bdf9a163b)

Reply:
I am getting this:

![image](https://github.com/user-attachments/assets/5b77e592-b529-43a3-a8ae-848b7055a0d8)

Reply:
Also about the spacings, I will double check as we didn't have figma.

Reply:
I kinda just want to fix the playground here. What you think?
1. Option - We just fix this in the playground. If the merchant has such rule we should follow what the merchant has.
2. Option - We make a defensive rule, but then we need to do it in the base css styles and not specifically for this component has this can happen again.

Reply:
I believe it is better to have a defensive rule on our side. It does not have to be global rule - I guess it is better to make the CSS of this Component more explicit in this case (by setting the font-weight instead of inheriting from somewhere else)

Reply:
Fixed.